smtp.yml

```diff
@@ -107,6 +136,18 @@ bind:
     required: true
     type: string
     default: ${request.instance_id}
+  - name: bounce_topic_arn
+    default: ${instance.details["bounce_topic_arn"]}
+    overwrite: true
+    type: string
+  - name: complaint_topic_arn
+    default: ${instance.details["complaint_topic_arn"]}
+    overwrite: true
+    type: string
+  - name: delivery_topic_arn
+    default: ${instance.details["delivery_topic_arn"]}
```

Comment:
currently debugging an issue with this line

`error generating bind variables: 1 error(s) occurred: couldn't compute the value for \"delivery_topic_arn\", template: \"${instance.details[\\\"delivery_topic_arn\\\"]}\", key \"delivery_topic_arn\" does not exist in map instance.details`